lib/diego/client.rb

```diff
@@ -175,6 +175,7 @@ def with_request_error_handling(&blk)
       tries ||= 3
       yield
     rescue => e
+      @cached_active_bbs_id = current_active_bbs_id
```

Comment:
Here I would set `@cached_active_bbs_id` to `nil`. Then the next call to `active_bbs_id` gets the current one.

Reply:
Yep, agreed, that's better